ts/Dashboards/Components/KPIComponent/KPIComponent.ts

```diff
@@ -405,6 +406,118 @@ class KPIComponent extends Component {
         this.chart?.destroy();
         super.destroy();
     }
+
+    /**
+     * Calculates value sum based on the provided values.
+     *
+     * @param values
+     * All particular column values.
+     *
+     * @returns
+     * The calculated sum value.
+     *
+     * @internal
+     */
+    private calculateSumValue(values: number[]): number {
+        let calculatedValue = 0;
+
+        for (let i = 0; i < values.length; i++) {
```

Comment:
`for (let i = 0, iEnd = values.length; i < iEnd; ++i) {` - is faster and more efficient